### DISCUSSION TITLE
holiday approval

### DISCUSSION THREAD

**Discussion by pkoevesdi** [kimai/Paid-plugins#176] [Questions & Answers] [answered]
Is there a function I didn't find to approve holiday requests?
Like: Worker 1 wants holiday, worker 2 (project manager) has to approve, before it is booked and worker 1 can go.

**Comment by kevinpapst** [comment, marked as answer]
No, approval is on the roadmap, check Roadmap on shop page: 
https://www.kimai.org/store/controlling.html
